mix.exs

```diff
@@ -64,7 +64,8 @@ defmodule NervesTime.MixProject do
       {:muontrap, "~> 0.5"},
       {:elixir_make, "~> 0.6", runtime: false},
       {:dialyxir, "~> 1.0.0", only: [:dev, :test], runtime: false},
-      {:ex_doc, "~> 0.22", only: :docs, runtime: false}
+      {:ex_doc, "~> 0.22", only: :docs, runtime: false},
+      {:tzdata, "~> 1.1"}
```

Comment:
tzdata is a big dependency for some Nerves users, so I'm hoping that there's a way to avoid pulling it in. It seems like it's only needed to validate the timezone name?

Reply:
That's correct.

Reply:
I'll relocate that test to Piston and have Piston depend on Tzdata.